beekeeper-api/src/test/java/com/expediagroup/beekeeper/api/response/HousekeepingMetadataResponseTest.java

```diff
@@ -46,6 +46,8 @@ public void testConvertToHouseKeepingMetadataResponsePage() {
 
     assertThat(metadataResponsePageList.get(0)).isEqualTo(metadataResponse1);
     assertThat(metadataResponsePageList.get(1)).isEqualTo(metadataResponse2);
+    assertThat(metadataResponsePage.getTotalElements()).isEqualTo(2L);
+    assertThat(metadataResponsePage.getTotalPages()).isEqualTo(1L);
```

Comment:
add another test where these values are not the default ones (more than 1 page and total num of elements is higher than the page size)